pkg/selinux/manifest.json.in

```diff
@@ -6,8 +6,7 @@
 
     "tools": {
         "setroubleshoot": {
-            "label": "SELinux",
-            "order": 8
+            "label": "SELinux"
```

Comment:
Does a trailing comma matter here to have fewer changes? Nitpick though, doesn't really make a difference.